Sealer/sealer.cc

```diff
@@ -603,38 +602,48 @@ void kRun(const ConnectPairsParams& params,
 
 	printLog(logStream, "Flanks inserted into k run = " + IntToString(flanks.size()) + "\n");
 
-	for (read1_it = flanks.begin(); read1_it != flanks.end();) {
-		success = false;
+	int counter = 0;
+	vector<map<FastaRecord, map<FastaRecord, Gap> >::iterator> flanks_closed;
+#pragma omp parallel private(read1_it, read2_it) firstprivate(counter)
+	for (read1_it = flanks.begin(); read1_it != flanks.end(); read1_it++) {
```

Comment:
Please use preincrement `++read1_it` for style (and theoretical performance benefit, though probably not actual).

Reply:
For storing keys in a vector, it seems it would be possible to fully replace the map there with a vector in the first place, while building flanks (Actually, is that what you meant?). Otherwise, it might be a bit slow to transfer keys to a vector.